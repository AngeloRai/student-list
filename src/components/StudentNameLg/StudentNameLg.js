const StudentNameLg = ({ name }) => {
  return (
    <div className="font-extrabold text-3xl" style={{ fontWeight: "900" }}>
      {name}
    </div>
  );
};

export default StudentNameLg;
