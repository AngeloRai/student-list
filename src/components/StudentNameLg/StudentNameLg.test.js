import React from "react";
import { shallow, mount } from "enzyme";
import StudentNameLg from "./StudentNameLg";
import toJson from "enzyme-to-json";

describe("tests student name component", () => {
  const mockName = "John";

  it("renders without crashing", () => {
    const wrapper = shallow(<StudentNameLg name={mockName} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("accepts props", () => {
    const wrapper = mount(<StudentNameLg name={mockName} />);
    expect(wrapper.props().name).toBe(mockName);
  });
});
