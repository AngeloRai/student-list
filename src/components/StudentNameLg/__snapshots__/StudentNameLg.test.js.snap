// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests student name component renders without crashing 1`] = `
<div
  className="font-extrabold text-3xl"
  style={
    Object {
      "fontWeight": "900",
    }
  }
>
  John
</div>
`;
